case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+---
+title: Azure DevOps Case Study
+keywords: development platforms, Azure DevOps, Microsoft
+tags: Development Platforms, Azure DevOps, Microsoft
+sidebar: development-platforms
+permalink: development-platforms/case-study-azure-devops.html
+folder: development-platforms
+---
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [Brief History](#brief-history)
+3. [Features](#features)
+4. [Strengths](#strengths)
+5. [Weaknesses](#weaknesses)
+6. [Pricing](#pricing)
+7. [Comparison](#comparison)
+8. [How To](#how-to)
+9. [Summary](#summary)
+10. [Credits](#credits)
+11. [References](#references)
+
+## **Introduction**
+
+<div style="width: 600px;">
+
+![Azure DevOps offerings](https://miro.medium.com/max/800/1*8orwInnxqPRhrcKf9aOo9Q.png)
+
+<p style="padding: 0; margin: 0;">
+
+*Source: google*
+
+</p>
+
+</div>
+
+**Azure DevOps** was initially released in 2018 by Microsoft. It is a set of developer tools, services, and features that enable teams to plan, develop, test, and deliver software more effectively. Azure DevOps integrates with the broader Azure cloud computing platform, enabling users to take advantage of its features and services, such as continuous integration and deployment, and virtual machines. Azure DevOps has since become one of the leading development platforms and is widely used by development teams across the world.
+
+Azure DevOps is for the most part free to use, however there is some limitations with using only a free tier option. I will go in deeper details about the pricing [here](#pricing)
+
+## **Brief History**
+
+In 2005 the service that we today refer to as Azure DevOps was named Team Foundation server(TFS). It was released as a set of development tools and services for teams building software applications. After its release in 2005 it has evolved and expanded with new features and services and have quickly become the most popular development platforms for Microsoft-centric teams. TFS has been around for more than a decade, and it has evolved a lot since its inception in 2005. There are professionals in the industry whose entire careers have been dedicated to managing TFS. Such expertise has been necessary because it was quite complicated to administer.
```

Comment:
```suggestion
In 2005 the service that we today refer to as Azure DevOps was named Team Foundation server (TFS). It was released as a set of development tools and services for teams building software applications. After its release in 2005 it has evolved and expanded with new features and services and have quickly become the most popular development platforms for Microsoft-centric teams. TFS has been around for more than a decade, and it has evolved a lot since its inception in 2005. There are professionals in the industry whose entire careers have been dedicated to managing TFS. Such expertise has been necessary because it was quite complicated to administer.
```

Please make sure that (brackets) are followed by a space. You can do this in VS Code with find and replace + regex:

Regex pattern: `\)(?![\s.])`
Replace with: `) `

<img width="530" alt="image" src="https://user-images.githubusercontent.com/10801540/218657710-b99151f1-5032-4c56-929e-4ffeb4a0510e.png">